pkgs/tools/security/magnifier/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, llvmPackages
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "magnifier";
+  version = "unstable-2022-08-26";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = "magnifier";
+    rev = "27c849db162e509aa413bdd971d7c55e32bc00a3";
+    sha256 = "sha256-yL71f3j9R52c1rFAsB9jPUoBASwdpyFeLSvSAAn9xms=";
+  };
```

I've noticed that magnifer also exposes a public library but your derivation doesn't provide [include](https://github.com/trailofbits/magnifier/tree/master/include/magnifier) files. Could you also please install development files in a separate `dev` output.

```suggestion
  };
  
  outputs = [ "out" "dev" ];
```